Sources/Phakchi/Encodable.swift

```diff
@@ -1,52 +1,63 @@
 import Foundation
 
-typealias JSONObject = [String: AnyObject]
+typealias JSONObject = [String: PactEncodable]
 
 public protocol PactEncodable {
-    var pactJSON: AnyObject { get }
+    var pactJSON: PactEncodable { get }
```

Comment:
@vincentisambart OK, I think `Interaction`, `Request`, `Response` shouldn't implement `PactEncodable` rather than `pactJSON` returns `PactEncodable`.  https://github.com/cookpad/Phakchi/pull/2/commits/c1fbc303ad783e1194c4555ab8e6b2be842f7a63